tello Test/tello_web_server.py

```
@@ -651,15 +740,672 @@ def _log_operation(self, operation: str, details: Dict[str, Any]):
+            
+            for i, ffmpeg_cmd in enumerate(ffmpeg_configs):
+                logger.info(f"FFmpeg設定 {i+1} を試行中...")
+                
+                try:
+                    # FFmpegプロセスを開始
+                    self.ffmpeg_process = subprocess.Popen(
+                        ffmpeg_cmd,
+                        stdout=subprocess.PIPE,
+                        stderr=subprocess.PIPE,
+                        bufsize=10**6  # バッファサイズを調整
+                    )
```

Pylint (3.3.7)

[refactor] Consider using 'with' for resource-allocating operations

(R1732)